### DISCUSSION TITLE
Can't connect to broker.emqx.io with host mqtt:// and port 1883 MQTT.js onError trigger, Error: connect ECONNREFUSED 54.244.173.190:1883

### DISCUSSION THREAD

**Discussion by Thinhle2701** [emqx/emqx#11974] [General]
I have used MQTTX for 3 months. It was really good to apply mqtt to my project IOT. However, I have problem with MQTTX. I cant connect to broker.emqx.io with host mqtt:// and port 1883 in MQTTX application on windows

When I see log, I saw that It can not connect to broker.emqx.io because "connect fail, MQTT.js onError trigger, Error: connect ECONNREFUSED 54.244.173.190:1883" at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)

I think it maybe from broker.emqx.io server with Port 1883 because I still use websocket to connect broker.emqx.io with Port 8083

Please help me solve this problem

![image](https://github.com/emqx/emqx/assets/59316091/dc0515f4-6e49-4e0c-b8c5-30917f16a3ad)

**Comment by id** [comment]
@Thinhle2701 please create the issue in the corresponding project https://github.com/emqx/mqttx.

**Comment by id** [comment]
@Thinhle2701 could you try again. We found out there was a problem with 1883 listener on the public broker.

**Reply by Thinhle2701** [reply]
It works !!
Thank you
